modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -187,6 +186,8 @@ CREATE TABLE IF NOT EXISTS items (
 
 To migrate the persistent data from one cluster to another, backup these two tables entirely from the source cluster and recreate them on the target one database. Do not to have applications deployed during this process on both clusters to prevent unexpected results.
 
+Note that if you are using a version of the Runtime Fabric agent higher than 2.6.22, the `id` parameter in the `CREATE TABLE IF NOT EXISTS items` statement no longer applies. 
```

Comment:
What if we phrase this the other way around? The schema with the modifications done in line 171 no longer contains the id definition.

Should the note state that for older versions than 2.6.22, an `id` column will be there for the items table?

Reply:
+1